### ISSUE TITLE
Extend icon resolver with attributes for role and localization

### ISSUE THREAD

**Issue by agitator** [plone/Products.CMFPlone#3402]
To fix https://github.com/plone/Products.CMFPlone/issues/3394 

The tag method in https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/browser/icons.py#L108 should handle aria related role info better. 

Also localization of alt tag would be useful.

**Comment by jensens** [comment]
Still relevant for Plone 6. Icon resolver lacks ARIA role and localization attributes. Linked issue #3394 also still open.
